ingredient/ingredient-databricks/src/plugin.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseIngredient, IngredientManager } from "@azbake/core"
+import { ARMHelper } from "@azbake/arm-helper"
+import ARMTemplate from "./arm.json"
+
+export class DataBricks extends BaseIngredient {
+
+    public async Execute(): Promise<void> {
```

Comment:
Consider adding diagnostic settings for any of the logs that may help with troubleshooting issues.
https://docs.microsoft.com/en-us/azure/databricks/administration-guide/account-settings/azure-diagnostic-logs